build.sbt

```diff
@@ -105,6 +105,10 @@ lazy val root = tlCrossRootProject
     bench,
     binCompatTest
   )
+  .settings(
+    name := "cats",
```

Comment:
A bit of bikeshedding (yeah, that again)...
I noticed that different modules have their names defined in different ways:
<details>

```
sbt:root> print name
lawsJS / name
	Cats laws
lawsJVM / name
	Cats laws
alleycatsCoreJVM / name
	Alleycats core
freeNative / name
	Cats Free
kernelLawsJVM / name
	Cats kernel laws
freeJVM / name
	Cats Free
algebraLawsJS / name
	Cats algebra laws
testkitJS / name
	testkit
algebraLawsJVM / name
	Cats algebra laws
alleycatsCoreNative / name
	Alleycats core
unidocs / name
	cats-docs
alleycatsLawsJS / name
	Alleycats laws
kernelNative / name
	Cats kernel
testkitNative / name
	testkit
algebraJVM / name
	Cats algebra
coreJVM / name
	Cats core
kernelJS / name
	Cats kernel
alleycatsLawsJVM / name
	Alleycats laws
algebraJS / name
	Cats algebra
kernelLawsJS / name
	Cats kernel laws
lawsNative / name
	Cats laws
coreJS / name
	Cats core
freeJS / name
	Cats Free
algebraLawsNative / name
	Cats algebra laws
testsJVM / name
	tests
algebraNative / name
	Cats algebra
kernelLawsNative / name
	Cats kernel laws
alleycatsLawsNative / name
	Alleycats laws
coreNative / name
	Cats core
testsNative / name
	tests
testsJS / name
	tests
kernelJVM / name
	Cats kernel
bench / name
	bench
testkitJVM / name
	testkit
alleycatsCoreJS / name
	Alleycats core
binCompatTest / name
	binCompatTest
name
	root
```

</details>

I wonder if it makes sense to stick to a particular naming scheme for `root` here, i.e. `Cats `, `Cats root`, `cats-root`, etc. Later, we could provide other modules with names chosen according to the same naming convention.